kingpin/actors/rightscale/server_array.py

```diff
@@ -705,6 +706,12 @@ def _execute(self):
         # Finally, monitor all of the tasks for completion.
         self.log.info('Queueing %s tasks' % count)
         task_waiting = []
+
+        import os
+        os.environ['TZ'] = 'UTC'
```

Comment:
Set this in a constant at the top of the file somewhere? Maybe even in the base actor?